build.gradle

```diff
@@ -149,15 +149,6 @@ license {
 	exclude('**/*.json') // JSON files do not have headers
 }
 
-task releaseJavadoc(type: Javadoc) {
-	title = project.property('version')
```

Comment:
the release javadocs task is needed cause it ensures the javadocs have the right title

Reply:
We can add the title with version to javadoc tasks, I removed the nightly dependsOn so it should be okay right?